test-suite/src/function/rand.rs

```diff
@@ -0,0 +1,38 @@
+use crate::*;
+
+test_case!(rand, async move {
+    use gluesql_core::prelude::{Payload, Value::*};
+
+    let test_cases = vec![
+        ("CREATE TABLE mytable (id int)", Ok(Payload::Create)),
+        (r#"INSERT INTO mytable VALUES (1)"#, Ok(Payload::Insert(1))),
+        (
+            "SELECT RAND() as r FROM mytable",
```

Comment:
`RAND` function without seed is not supposed to generate re-produceable result, but it looks that `RAND` always generate same values.

Writing tests for random functions, there exists a good reference.
`GENERATE_UUID`
https://github.com/gluesql/gluesql/blob/main/test-suite/src/function/generate_uuid.rs
https://github.com/gluesql/gluesql/blob/1e0f06f43990e26f9a506f5966ff4177b04b8f46/test-suite/src/function/generate_uuid.rs#L30-L34